### PR TITLE
메뉴 선택 후, 1space 공간 추가 및 copy-paste 시 에러 해결

### PR DESCRIPTION
**Pull request by gkswjdzz**

### CODE REVIEW

File: static/index.js [hunk 1]
```diff
@@ -15,16 +15,27 @@ const KEY_CODE = {"TAB" : 9, "UP" : 38, "DOWN" : 40, "ENTER" : 13, "PASTE" : 86}
 // Quill Text editor Initialize
 // *****************************
 var options = {
```
Comment (minsulee2) [inline static/index.js]: Should be const, init?

File: static/index.js [hunk 2]
```diff
@@ -15,16 +15,27 @@ const KEY_CODE = {"TAB" : 9, "UP" : 38, "DOWN" : 40, "ENTER" : 13, "PASTE" : 86}
...
-    theme: null
+    theme: null,
 };
 var quill = new Quill('.editor', options);
```
Comment (minsulee2) [inline static/index.js]: const as well?

File: static/index.js [hunk 3]
```diff
@@ -156,15 +167,16 @@ function complete(){
...
-    selection.deleteFromDocument();
-    selection.getRangeAt(0).insertNode(document.createTextNode(paste));
+// paste plane text only https://stackoverflow.com/questions/12027137/javascript-trick-for-paste-as-plain-text-in-execcommand
+editor.addEventListener('paste', (e) => {
+     // cancel paste
+     e.preventDefault();
 
-    event.preventDefault();
+     // get text representation of clipboard
+     var text = (e.originalEvent || e).clipboardData.getData('text/plain');
```
Comment (minsulee2) [inline static/index.js]: const as well

File: static/index.js (hunk 1 above)
Comment (gkswjdzz) [reply static/index.js]: I fixed it. thanks.

File: static/index.js (hunk 2 above)
Comment (gkswjdzz) [reply static/index.js]: I fixed it. thanks.

File: static/index.js (hunk 3 above)
Comment (gkswjdzz) [reply static/index.js]: I fixed it. thanks.

**Review by minsulee2 — APPROVED** [review]
LGTM!

**Pull request merged**